fe/src/components/Tab/HostTab.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-app class="pl-10">
+    <v-navigation-drawer permanent>
+      <v-list dense>
+        <v-list-item-group>
+
+          <!--Listings-->
+          <v-list-item href="#listings" @click="activeOption = 'l'" :class="{'option-active' : activeOption === 'l'}">
```

Comment:
Since `activeOption` is just 2 options we can change the type to boolean, please change the naming too to `isActive`.

```suggestion
          <v-list-item href="#listings" @click="function instead" :class="{'option-active' : activeOption}">
```

Reply:
Just did this ma'am 🙇